### ISSUE TITLE
Property 'compareSnapshotCommand' does not exist on type 'Chainable<undefined>'

### ISSUE THREAD

**Issue by edupeeth** [mjhea0/cypress-visual-regression#10]
**Following is my plugins/index.js**

```
const cucumber = require('cypress-cucumber-preprocessor').default;
const browserify = require('@cypress/browserify-preprocessor');
const getCompareSnapshotsPlugin = require('cypress-visual-regression/dist/plugin');


module.exports = (on, config) => {
    const options = browserify.defaultOptions;

    options.browserifyOptions.extensions.unshift('.ts');
    options.browserifyOptions.plugin.unshift(['tsify', { project: './cypress/tsconfig.json' }]);
    on('file:preprocessor', cucumber(options));
    getCompareSnapshotsPlugin(on);
}

```

**support/commands.js**

```
const compareSnapshotCommand = require('cypress-visual-regression/dist/command');
compareSnapshotCommand();
```

**cypress.json**

```
{
    "baseUrl": "http://localhost:4200",
    "testFiles": "**/*.feature",
    "video": false,
    "env": {
        "auth_username": "admin",
        "auth_password": "admin"
    },
    "screenshotsFolder": "cypress/snapshots/actual",
    "trashAssetsBeforeRuns": true
}
```
**Following is my feature file for test** -

```
Scenario: Screenshot Test
    Given Sample
```
This is what I have in test -

```
Given('Sample', () => {
    cy.compareSnapshot('home');
});
```

Error snapshot -

**Comment by mjhea0** [comment]
Per the README, use `cy.compareSnapshot()` not `cy. compareSnapshotCommand()`

**Comment by edupeeth** [comment]
Even cy.compareSnapshot() has same error.

**Comment by mjhea0** [comment]
which version of cypress are you using?

**Comment by edupeeth** [comment]
3.3.1

**Comment by mjhea0** [comment]
I've only tested up to v3.2.0. I'll see if I have time today to look into it.

**Comment by mjhea0** [comment]
Just tested on 3.3.1 and it works fine. Released v1.0.2.
